services/api-server/tests/unit/test_settings.py

```diff
@@ -14,7 +14,9 @@
 # bring .env-devel in here
 def test_min_environ_for_settings(monkeypatch):
     monkeypatch.setenv("WEBSERVER_HOST", "production_webserver")
-    monkeypatch.setenv("WEBSERVER_SESSION_SECRET_KEY", "REPLACE ME with a key of at least length 32.")
+    monkeypatch.setenv(
+        "WEBSERVER_SESSION_SECRET_KEY", "REPLACE ME with a key of at least length 32."
+    )
```

Comment:
one stupid question, what is the difference between monkeypatching and using the mock library? as far as I understand this monkeypatch fixture you can also replace functions right?

Reply:
they way i see it is that monkeypatch is more high-level in the sense that e.g. you can replace easily environment variables in a simple call while mock is more lower level

See examples here https://docs.pytest.org/en/latest/monkeypatch.html